app.js

```diff
@@ -117,6 +117,19 @@ app.use((req, res, next) => {
   next()
 })
 
+/**
+ * Used to append the apps version to the end of an asset declaration in a view
+ * Will invalidate older versions of that asset.
+ */
+app.use((req, res, next) => {
+  app.locals.assetUniqueId = () => {
+    const id = process.env.TAG_VERSION;
+    return id;
+  }
+  next()
+})
+
+
```

Comment:
This should go in the middleware folder, we've been pulling our custom middleware out of app.js to keep it from getting too big.